sdk/core/Azure.Core/src/Shared/HttpMessageSanitizer.cs

```diff
@@ -44,7 +44,12 @@ public string SanitizeUrl(string url)
                 return url;
             }
 
+#if NET5_0
```

Comment:
Why not `IndexOfOrdinal` here?

Reply:
Didn't want to make everyone include yet another shared source file for this single callsite.